wifiphisher/pywifiphisher.py

```diff
@@ -624,9 +630,13 @@ def cb(pkt):
                     if args.accesspoint not in [pkt.addr1, pkt.addr2]:
```

Comment:
Now if this is properly set to `False` as I recommend to another comment, we will skip this check and proceed with deauthing.